src/css/header.css

```diff
@@ -47,17 +42,21 @@
   display: none;
 }
 
+@media screen and (min-width: 375px) {
+  .header-container {
+    width: 335px;
+  }
+}
```

Comment:
delete this media rule, it must be rubber layout, but not adaptive

Reply:
According to our technical task the layout is rubber from 320px and becomes adaptive from 375px.